www/docs/migration/migrate-from-v9-to-v10.mdx

```diff
@@ -161,28 +161,27 @@ type GreetingInput = InferQueryInput<'greeting'>;
 
 #### v10
 
-```ts
-// The same helper is shipped out of the box.
-type GreetingInput = inferProcedureInput<AppRouter['greeting']>;
-```
-
-#### v10 inference helper
+#### Inference helpers
 
 ```ts twoslash
 // @module: esnext
 // @include: server
 // @filename: index.ts
 // ---cut---
-import type { GetInferenceHelpers } from '@trpc/server';
+// Inference helpers are now shipped out of the box.
+import type {
+  GetInputInferenceHelpers,
+  GetOutputInferenceHelpers,
+} from '@trpc/server';
 import type { AppRouter } from './server';
 
-export type AppRouterTypes = GetInferenceHelpers<AppRouter>;
-
-type PostCreate = AppRouterTypes['post']['create'];
+// @noErrors
```

Comment:
Is there a reason we need this?

Reply:
I added it because it was there in the previous example of the "inferring types" docs. Just removed it from the two new examples